### ISSUE TITLE
Why the api is unstable right now 

### ISSUE THREAD

**Issue by snowby666** [snowby666/poe-api-wrapper#138]
The api has been unstable for a couple of weeks. This is due to cloudflare protection with JS challenge when sending request (spoofed as robots/automatic/spamming requests) to www.quora.com/poe_api/gql_POST plus some redirecting methods. I've tried adding the cf_clearance token but it didnt work.

I'm pretty sure that poe.com also set up an aggressive rate limiter as well.

> **Solution?**
I'm working on a way using a headless browser to bypass this (currently testing with playwright). Maybe with addition of proxy_switcher for undetected connection. Any suggestion or further help is appreciated.

**Comment by FightingFranky** [comment]
The problem occurred more than before after I just sent less than 5 requests.

**Comment by noseon** [comment]
undetected-chromedriver to do some things and leave this part of your code in pyc compiled so that it doesn't know what you did. I have part of this made to run on Google Colab. If you want the code, I won't post it here for obvious reasons. Create an account on Telegram and post @ and I'll send you the code.

**Comment by bluusun** [comment]
You may also try pupeteer for Python including stealth plugins.

**Comment by AkideLiu** [comment]
no luck with even one single message request, got the following error :

```
RuntimeError: Rate limit exceeded for sending requests to poe.com. Please try again later.
```

**Comment by Ma-Yongqiang** [comment]
Run on colab

RuntimeError: Rate limit exceeded for sending requests to poe.com. Please try again later.

**Comment by snowby666** [comment]
This is fixed in the v1.4.3. Please check documentation for new config.
